fizzbuzz.py

```diff
@@ -0,0 +1,53 @@
+class Fizzbuzz:
+    def __init__(self, fb_list = None) -> None:
+        if fb_list is None:
+            fb_list = []
+        self.fb_list = fb_list
+
+    def fizzbuzz(self, num = 0):
+        if num < 0 or num > pow(10, 4):
+            raise ValueError('invalid number')
+        for i in range(1, num+1):
+            if i % 3 == 0 and i % 5 == 0:
+                self.fb_list.append('FizzBuzz')
+            elif i % 3 == 0 and i % 5 != 0:
+                self.fb_list.append('Fizz')
+            elif i % 3 != 0 and i % 5 == 0:
+                self.fb_list.append('Buzz')
+            elif i % 3 != 0 and i % 5 != 0:
+                self.fb_list.append(str(i))
```

Comment:
Your conditionals are overkill. The `i % 3` and `i%5` in the else's are redundant because the first condition already ensures that those cases are covered.

Reply:
I was trying to follow be explicit rather than implicit. How do I know if it's overkill? Just a feeling?